src/Shellbar/Shellbar.js

```diff
@@ -499,8 +504,11 @@ Shellbar.propTypes = {
     notifications: PropTypes.object,
     /** Holds product titles and navigation */
     productMenu: PropTypes.array,
-    /** For navigating between products */
-    productSwitch: PropTypes.object,
+    /** For navigating between products. An object that contains an accessible label for product switch button. */
+    productSwitch: PropTypes.shape({
+        /** Accessible label for product switch button */
+        label: PropTypes.string.isRequired
```

Comment:
👏  Should we mark this as a breaking change when merging since this is now required?

Reply:
Oh I'm sorry - it was always marked as required - I read the diff wrong 😞